### PR TITLE
Filter Nodes by Network

### PR DESCRIPTION
**Pull request by gertie-sheshe**
Add ability to see which networks a node belongs to.

Fixes [#176777141](https://www.pivotaltracker.com/story/show/176777141)

### Screenshots
networks-2

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -735,6 +764,21 @@ function showAllSelect() {
 	}
 }
 
+function showAllSelectNetwork() {
+	const custom_select_network = d3.select("#custom-select-network");
+	if (custom_select_network.select(".select-items-network").empty()) {
+		custom_select_network.append("div").attr("class", "select-items-network");
+		custom_select_network.select("select").selectAll("option").each(function (d) {
+			console.log(d3.select(this).html());
```
Comment (kilemensi) [inline src/js/index.js]:
```suggestion
```

**Review by kilemensi — COMMENTED** [review]
👍🏾 @gertie-sheshe 

## Issue

One real issue is selecting type "forgets" any network that was selected
Peek 2021-02-02 15-29

## Improvements

1. Selecting one dropdown should close the other if it's open (or clicking an open select should close it)
Peek 2021-02-02 15-30
2. Update networks as below:

```csv
AirNow 26
AirQO 24
OpenData Durban 28
PurpleAir 23
Sensor.Community 29
sensors.AFRICA Everything else
SmartCitizen 27
```
i.e. Remove OpenAQ for now.

**Review by nerdlet — APPROVED** [review]
@gertie-sheshe LGTM!

File: src/js/index.js [hunk 2]
```diff
@@ -735,6 +764,21 @@ function showAllSelect() {
 	}
 }
 
+function showAllSelectNetwork() {
```
Comment (KhadijaMahanga) [inline src/js/index.js:792]: @gertie-sheshe we've missed setting the arrow to active at this point. Its always pointing upwards  i.e `inactive`

**Review by KhadijaMahanga — COMMENTED** [review]
👍🏾

File: src/js/index.js [hunk 3]
```diff
@@ -749,3 +793,15 @@ function switchTo(element) {
 	reloadMap(user_selected_value);
 	custom_select.select(".select-items").remove();
 }
+
+function switchToNetwork(element) {
+	const custom_select_network = d3.select("#custom-select-network");
+	custom_select_network.select("select").property("value", element.id.substring(12));
```
Comment (KhadijaMahanga) [inline src/js/index.js:828]: why are we `substring`'ing here?

Comment (gertie-sheshe) [reply src/js/index.js:828]: To get the value and update when it's selected @KhadijaMahanga

**Comment by gertie-sheshe** [comment]
@kilemensi OpenData Durban doesn't seem to return anything - should that be the case?

**Comment by kilemensi** [comment]
> @kilemensi OpenData Durban doesn't seem to return anything - should that be the case?

English please @gertie-sheshe ... are you asking if it's okay not to have Open Data Durban nodes or?

**Comment by gertie-sheshe** [comment]
😅 @kilemensi yes. There are no nodes under the OpenData Durban network, so nothing shows. Wondered if that's alright

**Comment by gertie-sheshe** [comment]
> > @kilemensi OpenData Durban doesn't seem to return anything - should that be the case?
> 
> English please @gertie-sheshe ... are you asking if it's okay not to have Open Data Durban nodes or?

😅 @kilemensi yes. There are no nodes under the OpenData Durban network, so nothing shows. Wondered if that's alright

**Comment by kilemensi** [comment]
> > > @kilemensi OpenData Durban doesn't seem to return anything - should that be the case?
> > 
> > 
> > English please @gertie-sheshe ... are you asking if it's okay not to have Open Data Durban nodes or?
> 
> sweat_smile @kilemensi yes. There are no nodes under the OpenData Durban network, so nothing shows. Wondered if that's alright

Ahh... That's alright, the sensors team will need to login into the dashboard and assign some nodes to it. As long as we have it represented in the map, our work is done.

**Comment by KhadijaMahanga** [comment]
@kilemensi @gertie-sheshe there are nodes under opendata-durban https://api.sensors.africa/admin/sensors/node/?owner__id__exact=28 but usually it has a day before data. So if we're pulling nodes using `v2/nodes` we will never see them

**Pull request merged**